lib/debugger/widgets/selectionList.lua

```diff
@@ -76,15 +107,17 @@ return function(Plasma)
 		end)
 
 		Plasma.useEffect(function()
-			refs.button.TextLabel.Text = text
-			refs.button.Icon.Text = icon or ""
-			refs.button.Icon.Visible = not not icon
+			refs.button.container.TextLabel.Text = text
+			refs.button.container.Icon.Text = icon or ""
+			refs.button.container.Icon.Visible = not not icon
 		end, text, icon)
 
-		refs.sideText.Visible = not not sideText
-		refs.sideText.Text = sideText or ""
-		refs.sideText.TextColor3 = if selected then style.textColor else style.mutedTextColor
-		refs.button.TextLabel.TextTruncate = sideText and Enum.TextTruncate.AtEnd or Enum.TextTruncate.None
+		refs.button.container.sideText.Visible = not not sideText
```

Comment:
```suggestion
		refs.button.container.sideText.Visible = sideText ~= nil
```
the `not not`'s are hard to parse what the actual intent is

Reply:
Sure. These expressions were brought over from old code.

Reply:
Too clever for our own good